lib/aiken/rats.ak

```diff
@@ -0,0 +1,88 @@
+// Rationals
+// Authors:
+// Micah Kendall
+// <other authors go here>
```

Comment:
We do appreciate you very much, but we don't have the habits of crediting authors on top of source files. Git fills that role already :)